src/main/webapp/app/styles/app.scss

```diff
@@ -4,7 +4,7 @@
 // @import 'node_modules/bootstrap/scss/bootstrap';
 // @import '~bootswatch/dist/materia/bootswatch';
 .bg-image {
-  background-image: url('./bounties_bg.jpg');
+  background-image: url('../assets/bounties_bg.jpg');
```

Comment:
Most media like images, svgs etc... are in a folder named content. So I dont know, which is better, "content" or "assets" but either way it makes more sense if they are all in thesame location. Agreed?

Reply:
Agreed. Here i will put just logos, and images. Since u have trouble loading them from content/images

Reply:
It's not that the app couldn't load from content, I just had challenges referencing the path. But that's ok.